### PR TITLE
[Memgraph 2.17 < ]Update Kubernetes docs 

### PR DESCRIPTION
**Pull request by antejavor**
### Description

Please briefly explain the changes you made.

### Pull request type

Please check what kind of PR this is:

- [x] Fix or improvement of an existing page
- [ ] New documentation page, release related

### Related PRs and issues

PR this doc page is related to: 
https://github.com/memgraph/helm-charts/pull/22

### Checklist:

- [x] Check all content with Grammarly
- [x] Perform a self-review of my code
- [x] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [x] The build passes locally
- [x] My changes generate no new warnings or errors
- [x] Add a corresponding label
- [x] If release-related, add a product and version label
- [ ] If release-related, add release note on product PR

### CODE REVIEW

**Comment by antejavor** [comment]
If release-related, add a product and version label cc @kgolubic. I am not sure what goes on here, so let me know if I did everything as I should.

File: pages/getting-started/install-memgraph/kubernetes.mdx [hunk 1]
```diff
@@ -7,27 +7,26 @@ import { Callout } from 'nextra/components'
...
+
+- [**Helm chart for Memgraph Lab**](#helm-chart-for-memgraph-lab)
+
+For details on the implementation of the Helm charts, you can go to [**Memgraph
+Helm charts repository**](https://github.com/memgraph/helm-charts). Due to
+numerous possible use cases and deployment setups via Kubernetes, the provided
+Helm charts are a starting point you can modify according to your needs. 
+
+This docs page contains the basic information on how to deploy Memgraph or
+Memgraph LAB on Kubernetes using Helm charts.
```
Comment (katarinasupe) [inline pages/getting-started/install-memgraph/kubernetes.mdx:29]:
```suggestion
Memgraph Lab on Kubernetes using Helm charts.
```

File: pages/getting-started/install-memgraph/kubernetes.mdx [hunk 2]
```diff
@@ -134,3 +135,151 @@ and log directories, it is necessary that `runAsUser` is set to `0` in the
...
+```yaml
+env:
+  - name: QUICK_CONNECT_MG_HOST
+    value: memgraph
+  - name: QUICK_CONNECT_MG_PORT
+    value: "7687"
+```
+Check all supported environment variables from the Memgraph website, [Data
+Visualization
+Section](https://memgraph.com/docs/data-visualization/install-and-connect#environment-variables)
```
Comment (katarinasupe) [inline pages/getting-started/install-memgraph/kubernetes.mdx:282]:
```suggestion
Check all supported environment variables from the Memgraph Lab documentation(https://memgraph.com/docs/data-visualization/install-and-connect#environment-variables).
```

**Review by katarinasupe — CHANGES_REQUESTED** [review]

File: pages/getting-started/install-memgraph/kubernetes.mdx (hunk 2 above)
Comment (antejavor) [reply pages/getting-started/install-memgraph/kubernetes.mdx]: Updated.

**Review by katarinasupe — APPROVED** [review]

**Comment by antejavor** [comment]
@kgolubic Once you take a look at this PR if it is ok for you, feel free to merge it.

**Pull request merged**